src/Main.java

```diff
@@ -0,0 +1,11 @@
+public class Main {
+
+    public static void main(String[] args) {
+
+	    HourlyRateEmployee john = new HourlyRateEmployee("John");
+	    FixedMonthlyRateEmployee jack = new FixedMonthlyRateEmployee("Jack");
+	    System.out.println("John's hourly salary is " + john.salary());
+	    System.out.println("Jack's fixed montly salary is " + jack.salary());
```

Comment:
If you store an emloyee name in variable - why you don't use it in **sout**? Like
john.getName() + "'s hourly salary is " + john.salary()